test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -193,6 +194,7 @@ public ClassConfiguration getValueClassUsedInCode() {
 		String superName = config.getSuperName();
 		String[] fields = config.getFields();
 		int extraClassFlags = config.getExtraClassFlags();
+		int classFileVersion = 61;
```

Comment:
A comment here would be helpful noting this corresponds to Java 17.

Reply:
Comment added.